docs/topics/proc_installing-launcher-tool-addon.adoc

```diff
@@ -15,6 +15,14 @@ link:https://docs.openshift.org/latest/minishift/using/addons.html[Add-ons] enab
 
 .Procedure
 
+. Ensure you are running a minishift instance which has at least 4 GB memory to run launcher
+
```

Comment:
can you add a `+` here to make sure the source block is included under the number list item